sklearn/ensemble/stacking_classifier.py

```diff
@@ -0,0 +1,319 @@
+            # Parallel fit each estimator
+
+            self.estimators_ = Parallel(n_jobs=self.n_jobs)(
+                    delayed(_parallel_fit)(clone(clf),
+                                           X, transformed_y, kwargs)
+                    for _, clf in self.estimators)
+            scores = self._est_predict(X)
+        else:
+            # Use the n_jobs of cross_val_predict
+            self.estimators_ = []
+            scores = []
+            for _, clf in self.estimators:
+                s1 = cross_val_predict(clf, X, y, cv=self.cv,
+                                       method=self._method(clf),
+                                       fit_params=kwargs,
+                                       verbose=self.verbose,
+                                       n_jobs=self.n_jobs)
+                s1 = self._form_meta_inputs(clf, s1)
+                scores.append(s1)
+                self.estimators_.append(clf.fit(X, y, **kwargs))
+            scores = np.concatenate([s.reshape(-1, 1) if s.ndim == 1 else s
+                                     for s in scores], axis=1)
+        self.meta_estimator_ = clone(self.meta_estimator)
+        self.meta_estimator_.fit(scores, transformed_y, **kwargs)
+        return self
+
+    def _form_meta_inputs(self, clf, predicted):
+        if self._method(clf) in ['predict_proba', 'predict_log_proba']:
+            # Remove first column to avoid multicollinearity since sum of
+            # probabilities equals 1
+            predicted = predicted[:, 1:]
+        if self._method(clf) == 'predict_log_proba':
+            # Replace inf
+            predicted = np.clip(predicted, -1e30, 1e30)
+        return predicted
+
+    def _est_predict(self, X):
+        """ Generate input to meta_estimator from predictions of estimators
+        """
+        predicted = []
+        for clf in self.estimators_:
+            s1 = getattr(clf, self._method(clf))(X)
+            s1 = self._form_meta_inputs(clf, s1)
+            predicted.append(s1)
+        return np.concatenate([s.reshape(-1, 1) if s.ndim == 1 else s
+                               for s in predicted], axis=1)
+
+    @if_delegate_has_method(delegate=('meta_estimator_', 'meta_estimator'))
+    def predict(self, X):
+        """Predict class labels for X.
+
+        Only available if fitted and ``meta_estimator`` supports ``predict``.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+
+        Returns
+        -------
+        C : array, shape = [n_samples]
+            Predicted class label per sample.
+        """
+        check_is_fitted(self, ['estimators_', 'meta_estimator_'])
+        predicted = self.meta_estimator_.predict(self._est_predict(X))
+        return self.le_.inverse_transform(predicted)
+
+    @if_delegate_has_method(delegate=('meta_estimator_', 'meta_estimator'))
+    def predict_proba(self, X):
+        """ Return probability estimates for the test vector X.
```

Comment:
PEP257 says the description should be right next to the opening quotes. please remove the leading space